src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/ITensor.cs

```diff
@@ -15,25 +20,104 @@ public interface ITensor<TSelf, T>
         // It looks like C#/.NET currently hits limitations here as it believes TSelf and T could be the same type
         // Ideally we could annotate it such that they cannot be the same type and no conflicts would exist
 
+        /// <summary>
+        /// Creates a new tensor with the specified lengths. If <paramref name="pinned"/> is true the underlying buffer is
```

Comment:
Everything except the first sentence should go in remarks.

Reply:
Fixed.